fstab.hardware

```diff
@@ -1,8 +1,8 @@
 # Android fstab file.
 
 #<src>                                                  <mnt_point>            <type>  <mnt_flags and options>                              <fs_mgr_flags>
-system                                                  /system                ext4    ro,barrier=1                                         wait,slotselect,avb=vbmeta_system,logical,first_stage_mount
-vendor                                                  /vendor                ext4    ro,barrier=1                                         wait,slotselect,avb=vbmeta,logical,first_stage_mount
```

Comment:
Why is the vendor line being changed?

Reply:
Because this is (kind of) a squahed revert of 2 commits
Please see the blame on this file to see how the original was before and after enabling chained partitions

Reply:
Actually I think we don't need avb=vbmeta for system either

Reply:
Are you sure it isn't required to have avb=vbmeta now? Why would they be doing it if it's not required?

Reply:
From what I have seen on other devices, avb=vbmeta is left as is for system but other partitions just use avb

Maybe we should set both to just avb and see if it still works